Makefile

```diff
@@ -52,6 +52,7 @@ export SELF_SIGNED_CERTS   ?= true
 # Setting the INSTALLATION_TYPE to managed-api will configure the values required for RHOAM installs
 export INSTALLATION_TYPE   ?= managed
 
+export ALERT_SMTP_FROM ?=no-reply-alert@devshift.org
```

Comment:
I think this should be `noreply-alert@devshift.org` - that's what it is in MTR

Reply:
Yap I agree, I will make any change for that.